src/main/java/com/redhat/devtools/lsp4ij/usages/LSPUsageSearcher.java

```diff
@@ -74,13 +78,19 @@ public void processElementUsages(@NotNull PsiElement element, @NotNull Processor
                 return;
             }
 
+            // Make sure that the search scope is respected when adding usages
+            SearchScope searchScope = options.searchScope;

```

Comment:
I noticed that the provided search scope wasn't being used to filter usages, so I fixed that.